iModelCore/ECPresentation/Tests/NonPublished/Unit/Content/ContentDescriptorTests.cpp

```diff
@@ -43,7 +43,7 @@ struct ContentDescriptorTests : ::testing::Test
 
     ContentDescriptor::CalculatedPropertyField* CreateCalculatedField(Utf8CP name) const
         {
-        auto field = new ContentDescriptor::CalculatedPropertyField(m_category, name, name, "", nullptr);
+        auto field = new ContentDescriptor::CalculatedPropertyField(m_category, name, name, Utf8String(""), nullptr);

```

Comment:
I suppose passing `""` doesn't work because the argument's type is `Nullable<Utf8String>`? Requiring to add those value strings to `Utf8String` is pretty painful... maybe we could add an override that takes care of this?

```cpp
void SetValue(Utf8CP value) { m_value = value ? Utf8String(value) : nullptr; InvalidateHash(); }
```

Reply:
Yes, that was the reason why `""` did not work. Changed `SetValue(...)` to your suggestion

Reply:
Could you please also revert the changes where you wrapped strings with `Utf8String(...)`?

Reply:
Changed `CalculatedPropertyField` similar to `SetValue`